VoltageDropCalculator.py

```diff
@@ -0,0 +1,92 @@
+# explain the relationship between the key and value by naming this dictionary properly
+# for example, a key of 10 (awg) corresponds to the value 10380 (circmils)
+awg_to_circmils = {
+  10: 10380,
+  8: 16510,
+  6: 26240,
+  4: 41740,
+  3: 52620,
+}
+
+# think about how these functions differ from the dictionary above.
+# in this case they perform a strict lookup as well. when might you use one or the other?
+# would you use a function or dictionary to compute a bit of arithmetic?
+# - generally speaking it seems a dictionary is useful for a strict lookup,
+# - whereas a function is likely used for continuous values derived from calculations.
+# what if you had a large number of values that correspond 1:1 with a pre-computed output?
+# what happens in the following functions when the input doesn't correspond to an output?
+# what happens in the dictionary when a given key is not present?
+# - both would throw an error? can I loop back through, using 'break' and 'continue'?
```

Comment:
The functions here currently don't throw an error, though possibly they should. It depends on if you want your program to exit when an invalid operation occurs, or you'd like to provide a retry mechanism. If you're going for retry you might consider an approach like this: https://wiki.python.org/moin/WhileLoop

Reply:
Back to my Codecademy interpreter - I was getting errors for invalid inputs. I will play around with the while loop and let you know what I come up with. Thanks for the input!